src/sctp_transport/mod.rs

```diff
@@ -333,6 +338,51 @@ impl RTCSctpTransport {
         self.state.load(Ordering::SeqCst).into()
     }
 
+    pub(crate) async fn collect_stats(
+        &self,
+        collector: &Arc<Mutex<StatsCollector>>,
+        worker: Worker,
+        peer_connection_id: String,
+    ) {
+        let dtls_transport = self.transport();
+
+        // TODO: should this be collected?
+        dtls_transport
+            .collect_stats(collector, worker.clone())
+            .await;
+
+        // data channels
+        let mut data_channels_closed = 0;
+        let data_channels = self.data_channels.try_lock().unwrap();
+        for data_channel in &*data_channels {
+            match data_channel.ready_state() {
+                RTCDataChannelState::Connecting => (),
+                RTCDataChannelState::Open => (),
+                _ => data_channels_closed += 1,
+            }
+            data_channel.collect_stats(collector, worker.clone()).await;
+        }
+
+        let mut reports = vec![];
+        reports.push(PeerConnection(PeerConnectionStats::new(
+            self,
+            peer_connection_id,
+            data_channels_closed,
+        )));
+
+        // conn
+        if let Some(_net_conn) = dtls_transport.conn().await {
+            let stats = ICETransportStats::new("sctp_transport".to_owned());
+            // TODO: get bytes out of Conn.
+            // bytes_received: conn.bytes_received,
+            // bytes_sent: conn.bytes_sent,
+            reports.push(SCTPTransport(stats));
+        }
```

Comment:
I don't see a way to get `bytes_received` or `bytes_sent`.

Reply:
I see that `DTLSConn` in the `webrtc-dtls` crate has a private `conn`. Perhaps if that's made public and released, `bytes_received` and `bytes_sent` can be gotten out of that?

Another thought is `fn bytes_received() -> usize` and `fn bytes_sent() -> usize` can be added to the required implementation of `Conn` (assuming it's possible for all implementers to track). That seems like it would require deeper changes, and perhaps multiple sub-crates to be released before it could be added to the stats code.